src/components/Quotes.jsx

```diff
@@ -0,0 +1,39 @@
+import { useState, useEffect } from 'react';
+
+const Quotes = () => {
+  const [quote, setQuote] = useState(null);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(false);
+
+  useEffect(() => {
+    const fetchQuote = async () => {
+      const response = await fetch('https://api.api-ninjas.com/v1/quotes?category=happiness',
+        {
+          headers: { 'X-Api-Key': 'VF/Pc9XBd+WWEkCjPg18aA==RDypQjoE00TFNCQq' },
```

Comment:
- [ ] [OPTIONAL] Kindly make sure to not expose any API keys in your code, you can use environment variables to add the API key securely to your code

Reply:
Thanks for the tip. Can you please walk me through implementing this or maybe point me in the direction of a relevant tutorial